docs/rfcs/1000-shamir-based-recovery.md

```diff
@@ -726,6 +727,121 @@ The claimer gets access to `reveal_token` and `data_key`, it can then retrieve `
 Then `ciphered_data` can be decrypted with `data_key`. From then on, the recovery works
 just like the recovery device system (see `parsec core import_recovery_device` CLI).
 
+## 4 - Delete a shamir setup
+
+Authenticated API:
+
+```json5
+[
+    {
+        "major_versions": [
+            4
+        ],
+        "req": {
+            "cmd": "shamir_recovery_deletion",
```

Comment:
```suggestion
            "cmd": "shamir_recovery_delete_setup",
```

The command name should contain a verb

(btw I guess `shamir_recovery_setup` is a bit weird on this regard given `setup` is both the verb and the noun, since after that we refer to the configuration set by this command as the `shamir setup`)

Reply:
Maybe `shamir_recovery_setup_setup` ?